### PR TITLE
Add battery level to protobuff message so it can be added to GUI

### PR DESCRIPTION
**Pull request by jsongCMU**
Protobuff from Khepera to server now has additional agent_status field, which currently only contains battery level data.

### CODE REVIEW

File: prog-template.c [hunk 1]
```diff
@@ -383,9 +383,9 @@ void UDPsendSensor(int UDP_sockfd, struct sockaddr_in servaddr, long double T, d
 
 	// Accelerometer
 	robosar_fms_Accelerometer proto_accel_data;
-	proto_accel_data.acc_x = velo_cmd.V; // send received lin vel cmd
-	proto_accel_data.acc_y = velo_cmd.W; // send received ang vel cmd
-	proto_accel_data.acc_z = override_flag;
+	proto_accel_data.acc_x = acc_X;
+	proto_accel_data.acc_y = acc_Y;
+	proto_accel_data.acc_z = acc_Z;
```
Comment (rachelzh9) [inline prog-template.c:424]: Was this previously set to the velocity commands instead for debugging purposes?

**Comment by rachelzh9** [comment]
pls resolve conflicts

**Review by rachelzh9 — CHANGES_REQUESTED** [review]

File: prog-template.c (hunk 1 above)
Comment (jsongCMU) [reply prog-template.c:424]: Yes, but the debugging has been resolved, so I wanted to set them back

**Review by rachelzh9 — APPROVED** [review]
lgtm

**Pull request merged**